torch/_refs/__init__.py

```diff
@@ -236,13 +243,33 @@ def __maybe_broadcast(x, shape):
 
 # TODO: add type promotion support
 def _make_elementwise_unary_reference(
-    prim: Callable, *, type_promotion_kind, aten_op=infer_aten_op, register_meta=False
+    prim: Callable,
+    *,
+    type_promotion_kind,
+    use_opmath,
+    CPU_use_opmath=None,
+    CUDA_use_opmath=None,
```

Comment:
why not just lower case here?

Reply:
yeah that'd be reasonable -- they're capitalized in a lot of the test suite today so I suppose I was thinking of it